cmd/bootstrap/cmd/constraints.go

```diff
@@ -39,6 +38,14 @@ func checkConstraints(partnerNodes, internalNodes []model.NodeInfo) {
 			partnerCONCount, internalCONCount, partnerCONCount*2+1)
 	}
 
+}
+
+// Checks constraints about the number of partner and internal nodes.
+// Internal nodes must comprise >2/3 of consensus committee.
+// Internal nodes must comprise >2/3 of each collector cluster.
+func checkCollectionConstraints(partnerNodes, internalNodes []model.NodeInfo) {
```

Comment:
⚠️ 

Please note that also Collector nodes are weighted. Here, we just count nodes. 
* In the previous implementation, this was no problem, because we checked that all nodes have equal stake. 
* Would suggest to move check https://github.com/onflow/flow-go/blob/fdbd24ca32980e04c5acb1d61aca48f4ca5cdc23/cmd/bootstrap/cmd/constraints.go#L17-L29 into this method here

Reply:
implemented in https://github.com/onflow/flow-go/pull/1371/commits/503e8eae901a323858140c152d7e2b3694e1307c and https://github.com/onflow/flow-go/pull/1371/commits/e011135fd2721458d04e7b84b928f296c2a9bb02